zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+This funding stream will start on expiry of the existing ``FS_FPF_ZCG`` funding stream [^zip-1015-funding-streams], and last for a further 1,260,000 blocks (approximately 3 years), ending at Zcash's 3rd halving.
+
+## Coin-holder-controlled Fund
+
+### One-time lockbox disbursal
+
+The coinbase transaction of the activation block of this ZIP MUST include an additional output to a 3-of-5 P2SH multisig with keys held by the Zcash Foundation, the Electric Coin Company, Shielded Labs, and two other organizations yet to be decided ("Key-holder Organizations").
+
+This output MUST follow the same consensus rules as if there were an additional ``FS_COINHOLDER_ONETIME`` funding stream lasting for a single block, with value for that block equal to the amount in the Deferred Dev Fund Lockbox at the activation height. (This value can be predicted in advance, given the activation height.) At the same time, the balance of the Deferred Dev Fund Lockbox will be reduced by that amount, i.e. reset to zero.
+
+Exactly one of the options in the next three sections will also be taken.
+
+### Option 1: Extend the lockbox funding stream
+
+This option extends the ``FS_DEFERRED`` lockbox funding stream by a further 1,260,000 blocks (approximately 3 years), ending at the 3rd halving.
+
+### Option 2: Additional funding stream
+
+An additional funding stream will be established, consisting of 12% of the block subsidy paid to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above. The Coin-holder-controlled Fund is considered to include both funds from the ``FS_COINHOLDER`` funding stream, and funds from the one-time lockbox disbursal described above.
+
+This stream will replace the existing ``FS_DEFERRED`` lockbox funding stream [^zip-1015-funding-streams]. That is, the end height of ``FS_DEFERRED`` will be changed to the activation height of this ZIP. The new ``FS_COINHOLDER`` stream will start at the same height and last until the 3rd halving.
+
+### Option 3: Periodic lockbox disbursement
+
+This would require a mechanism to pay from the Deferred Dev Fund Lockbox to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above, at periodic intervals. [TBD schedule and period.]
+
+### Requirements on use of the Coin-holder-controlled Fund
+
+The Key-Holder Organizations would be bound by a legal agreement to only use funds held in the Coin-holder-controlled Fund according to the specifications in this ZIP, expressed in suitable legal language. In particular, all requirements on the use of Deferred Dev Fund Lockbox funds in ZIP 1015 [^zip-1015-lockbox] MUST be followed for the funds obtained from the one-time disbursal.
+
+The Financial Privacy Foundation (FPF) would administer the Coin-holder-controlled Fund based on decisions taken by coin-holder voting, following a model decided by the community and specified in another ZIP [TBD].
```

Comment:
Note from Josh: 

>  FPF will not be involved. Some org (TBD) will need to be designated to conduct polling.

Reply:
This now reads "The Key-Holder Organizations would collectively administer the Coinholder-Controlled Fund based on decisions taken by coinholder voting, ..."